composer.json

```diff
@@ -27,9 +27,7 @@
     "require": {
         "php": ">=7.2",
         "cakephp/collection": "^4.0",
-        "cakephp/core": "^4.0",
         "cakephp/database": "^4.0",
-        "cakephp/datasource": "^4.0",
```

Comment:
why these removed ?

Reply:
Neither are directly depended upon and are just dependencies of dependencies.

Reply:
fair enough 
yes , they don't use directly in phinx , so good to remove